### PR TITLE
Adding code to print report dictionary from yaml block

### PR DESCRIPTION
**Pull request by devplayer55221**
1. Adding report block to local.yaml
2. Adding code for printing report block from local.yaml as dictionary with the command `mantis report -o example_org`

### CODE REVIEW

**Review by Prateek-Thakare — COMMENTED** [review]
Hi @devplayer55221 
Can you please revert the changes in the config_client.py and test the changes?

**Comment by devplayer55221** [comment]
Hi @Prateek-Thakare 
I reverted the changes to config_client.py.
Added report to the config_model.py.
It is working.

File: mantis/workflows/report_workflow.py
Comment (Prateek-Thakare) [inline mantis/workflows/report_workflow.py:1]: Can you directly use configprovider to access the config here? Having an util just for this is redundant. If any usecase arrives in future we can move this as an util

File: setup/docker/docker-compose.yml
Comment (Prateek-Thakare) [inline setup/docker/docker-compose.yml:1]: Please revert these changes to original and do not commit any changes on this file.

File: mantis/workflows/report_workflow.py
Comment (devplayer55221) [reply mantis/workflows/report_workflow.py:1]: Done. Changes removed from config_utils.py and modified report_workflow to directly use ConfigProvider

File: setup/docker/docker-compose.yml
Comment (devplayer55221) [reply setup/docker/docker-compose.yml:1]: Done.

**Comment by devplayer55221** [comment]
Thanks @Prateek-Thakare
